check_pihole.py

```diff
@@ -42,8 +42,8 @@ def main():
             message = "CRITICAL: "
         else:
             message = "WARNING: "
-    message = message + "Pi-hole is " + url_output["status"] + ": queries today - " + str(url_output["dns_queries_all_types"]) + ", domains blocked: " + str(url_output["ads_blocked_today"]) + ", percentage blocked: " + str(url_output["ads_percentage_today"])
+    message = message + "Pi-hole is " + url_output["status"] + ": queries today - " + str(url_output["dns_queries_all_types"]) + ", domains blocked: " + str(url_output["ads_blocked_today"]) + ", percentage blocked: " + str(url_output["ads_percentage_today"]) + "|queries=" + str(url_output["dns_queries_all_types"]) +" blocked=" +  str(url_output["ads_blocked_today"])
```

Comment:
It's actually from nagios plugin development, see [here](https://nagios-plugins.org/doc/guidelines.html)

It will look like [this ](https://i.imgur.com/v24lN0p.png) in icinga

Reply:
ah, sorry. missed some in last few years not using perf data in icinga2.